src/command-line/add.js

```diff
@@ -1,45 +1,67 @@
 "use strict";
 
-var ClientManager = new require("../clientManager");
-var colors = require("colors/safe");
-var program = require("commander");
-var Helper = require("../helper");
+const colors = require("colors/safe");
+const program = require("commander");
+const fs = require("fs");
+const Helper = require("../helper");
 
 program
 	.command("add <name>")
 	.description("Add a new user")
 	.action(function(name) {
-		var manager = new ClientManager();
-		var users = manager.getUsers();
-		if (users.indexOf(name) !== -1) {
-			log.error(`User ${colors.bold(name)} already exists.`);
+		if (!fs.existsSync(Helper.USERS_PATH)) {
+			log.error(`${Helper.USERS_PATH} does not exist.`);
 			return;
 		}
+
+		prompt(name);
+	});
+
+function prompt(name, callback) {
+	const ClientManager = require("../clientManager");
+
+	var manager = new ClientManager();
+	var users = manager.getUsers();
+
+	if (users.indexOf(name) !== -1) {
+		log.error(`User ${colors.bold(name)} already exists.`);
+		return;
+	}
+
+	log.prompt({
+		text: "Enter password:",
+		silent: true
+	}, function(err, password) {
+		if (!password) {
+			log.error("Password cannot be empty.");
+			return;
+		}
+
+		if (err) {
+			return;
+		}
+
 		log.prompt({
-			text: "Enter password:",
-			silent: true
-		}, function(err, password) {
-			if (!password) {
-				log.error("Password cannot be empty.");
+			text: "Save logs to disk?",
+			default: "yes"
+		}, function(err2, enableLog) {
+			if (err2) {
 				return;
 			}
-			if (!err) {
-				log.prompt({
-					text: "Save logs to disk?",
-					default: "yes"
-				}, function(err2, enableLog) {
-					if (!err2) {
-						add(
-							manager,
-							name,
-							password,
-							enableLog.charAt(0).toLowerCase() === "y"
-						);
-					}
-				});
+
+			add(
+				manager,
+				name,
+				password,
+				enableLog.charAt(0).toLowerCase() === "y"
```

Comment:
Not sure why we would do that. It was fine and `y` only before this PR, and most tools (like Git) will do this too.